src/components/Buttons/SendButton.js

```diff
@@ -0,0 +1,24 @@
+import SendIcon from "../../assets/Send.png";
+import styled from "styled-components";
+
+const StyledSendButton = styled.div`
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  padding: 0.5rem;
+  font-size: 1rem;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const SendButton = () => {
+  return (
+    <StyledSendButton>
+      {" "}
```

Comment:
??

Reply:
if this is on purpose and not a mistake, whatever you want to achieve can be done with CSS